flow/flow.json

```diff
@@ -0,0 +1,29 @@
+{
+	"networks": {
+		"emulator": "127.0.0.1:3569",
+		"mainnet": "access.mainnet.nodes.onflow.org:9000",
+		"testnet": "access.devnet.nodes.onflow.org:9000"
+	},
+	"accounts": {
+		"emulator-account": {
+			"address": "f8d6e0586b0a20c7",
+			"key": "a2f983853e61b3e27d94b7bf3d7094dd756aead2a813dd5cf738e1da56fa9c17"
```

Comment:
It would be better to add the expected signature `algo` here with the key value (the corresponding public key would change depending on the algo). 
Here is an example https://developers.flow.com/tools/flow-cli/flow.json/configuration#advanced-format-1